droid-api/src/test/java/uk/gov/nationalarchives/droid/internal/api/DroidAPITestUtils.java

```diff
@@ -38,8 +38,14 @@
 
 public class DroidAPITestUtils {
     public static DroidAPI createApi() throws SignatureParseException {
-        Path signaturePath = Paths.get("../droid-results/custom_home/signature_files/DROID_SignatureFile_V96.xml");
-        Path containerPath = Paths.get("../droid-results/custom_home/container_sigs/container-signature-20200121.xml");
+        Path signaturePath = Paths.get("../droid-results/custom_home/signature_files/DROID_SignatureFile_V109.xml");
+        Path containerPath = Paths.get("../droid-results/custom_home/container_sigs/container-signature-20221102.xml");
+
+//        Path signaturePath = Paths.get("../droid-results/custom_home/signature_files/DROID_SignatureFile_V109.xml");
```

Comment:
Do we need this commented out code here?

Reply:
Well spotted, that was oversight on my part when I was trying out newest test data with a string of signature files